.changeset/sweet-singers-argue.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-catalog-backend-module-github': minor
+---
+
+Remove use of `EventBroker` and `EventSubscriber`.
```

Comment:
I think it'd be good to point to the events option as the replacement here again, in case people missed that last time around